pkg/portal/operator_feature_flags_test.go

```diff
@@ -0,0 +1,202 @@
+package portal
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/go-test/deep"
+	"github.com/gorilla/mux"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+func TestOperatorFeatureFlags(t *testing.T) {
+	dbOpenShiftClusters, _ := testdatabase.NewFakeOpenShiftClusters()
+
+	fixture := testdatabase.NewFixture().
+		WithOpenShiftClusters(dbOpenShiftClusters)
+
+	_, err := time.Parse(time.RFC3339, "2011-01-02T01:03:00Z")
+	if err != nil {
+		t.Error(err)
+	}
+
+	fixture.AddOpenShiftClusterDocuments(
+		&api.OpenShiftClusterDocument{
+			ID:  "11111111-2222-2222-2222-333333333333",
+			Key: "/subscriptions/11111111-2222-2222-2222-333333333333/resourcegroups/resourcegroupname/providers/microsoft.redhatopenshift/openshiftclusters/cluster1",
+			OpenShiftCluster: &api.OpenShiftCluster{
+				ID: "/subscriptions/11111111-2222-2222-2222-333333333333/resourcegroups/resourcegroupname/providers/microsoft.redhatopenshift/openshiftclusters/cluster1",
+				Properties: api.OpenShiftClusterProperties{
+					// ProvisioningState: api.ProvisioningStateSucceeded,
+					// CreatedAt:         parsedTime,
```

Comment:
nit: can this commented out code be deleted please?

Reply:
Yup Done!